core/src/main/java/org/gbif/pipelines/core/utils/AvroUtil.java

```diff
@@ -0,0 +1,26 @@
+package org.gbif.pipelines.core.utils;
+
+import org.gbif.pipelines.io.avro.ExtendedRecord;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import org.apache.avro.file.CodecFactory;
+import org.apache.avro.file.DataFileWriter;
+import org.apache.avro.specific.SpecificDatumWriter;
+
+public class AvroUtil {
+
+  /**
+   * Configure and create FileWriter with ExtendedRecord Format
+   *
+   * @return DataFileWriter with extendedRecord and appropriate configurations
+   */
+  public static DataFileWriter<ExtendedRecord> getExtendedRecordWriter(OutputStream os, int syncInterval,
```

Comment:
please add javadoc to each param since this can be considered as expert use of Avro libs